server/src/session/error.rs

```diff
@@ -0,0 +1,15 @@
+#[derive(Debug, thiserror::Error)]
+pub enum Error {
```

Comment:
あって損ないので`Clone`と`Copy`も

```suggestion
#[derive(Debug, Clone, Copy, thiserror::Error)]
pub enum Error {
```